src/domain/repositories/schedule.ts

```diff
@@ -5,4 +5,6 @@ export interface ScheduleRepository {
   create: (schedule: Schedule) => Promise<void>
   update: (schedule: Schedule) => Promise<void>
   isAvailable: (doctorId:string, startAt: Date, endAt: Date) => Promise<boolean>
+  findOpenSchedules: (doctorId: string) => Promise<any>
```

Comment:
```suggestion
  findOpenSchedules: (doctorId: string) => Promise<any>
```
Coloca a tipagem maninho, pra conseguirmos saber qual o tipo que retorna